prdoc/pr_8000.prdoc

```diff
@@ -0,0 +1,27 @@
+title: Optimize origin checks
+doc:
+- audience: Runtime Dev
+  description: Optimize origin checks, avoid cloning and conversion when not needed.
+crates:
+- name: snowbridge-pallet-system
+  bump: major
```

Comment:
is it really major?

Reply:
good catch! I think `patch` should work fine

Reply:
indeed, some crate still need major because they change the bounds on their implementation and it can break rust code.

But other crate are good, update in https://github.com/paritytech/polkadot-sdk/pull/8000/commits/3145f292ed097aaf9fbfdac9b8508ca64da22559